lux/extensions/ui/css/navbar.py

```diff
@@ -15,67 +15,100 @@ def add_css(all):
     # STYLING
     navbar = vars.navbar
     #
+    navbar.transition.duration_in = '0.6s'
+    navbar.transition.duration_out = '0.2s'
+    navbar.transition.duration = '0.8s'
+    navbar.collapse.background = '#eaeaea'
```

Comment:
Good, if it helps removed the fix to top option for now so the navbar does not stay at the top.
For this comment out line https://github.com/quantmind/lux/blob/dev/luxjs/nav/sidebar.js#L100